components/constellation/pipeline.rs

```diff
@@ -159,6 +159,9 @@ pub struct InitialPipelineState {
     /// The ID of the pipeline namespace for this script thread.
     pub pipeline_namespace_id: PipelineNamespaceId,
 
+    /// Pipeline visibility to be inherited
```

Comment:
Can we keep the relevant bits of the previous comment on the old `is_visible`?

Reply:
Yeah, forgot to paste my updated doc after copying back the old field (reverted to the old name to keep API changes to a minimum).